docs/cheaha/tutorial/pytorch_tensorflow.md

```diff
@@ -0,0 +1,133 @@
+# Anaconda Environment Tutorial for PyTorch and TensorFlow
+
+The below tutorial would show you steps on how to create an Anaconda environment, activate, and install libraries/packages for machine and deep learning (PyTorch and Tensorflow) using an Anaconda environment on Cheaha. There are also steps on how to access and use the Shell Command Line Interface (CLI), and using Jupyter Notebook's Graphical User Interface (GUI) to work with these Anaconda environments. There are detailed steps here to guide your creation of a [Jupyter Notebook job.](../open_ondemand/ood_layout.md#interactive-apps)
+
+## Installing Anaconda Environments using Cheaha's Command Line Interface (CLI)
+
+To access the termainal (shell), please do the following.
+
+1. Login to [rc.uab.edu](https://rc.uab.edu)
+
+2. Create a job on Cheaha using the Interactive Apps dropdown option.![!Interactive Apps Dropdown Menu](images/interactive_dropdown.png)
+
+3. Select Jupyter Notebook, and fill out the options, as per your project needs, then click Launch.  For more information on compute needs, and a guide for selecting the right options click [here](../job_efficiency.md#estimating-compute-resources). ![!Jupyter Launch Button](images/jupyter_launch.png)
+
+4. Click the Connect to Jupyter button ![!Connect to Jupyter Button](images/connect_to_jupyt_button.png)
+
+    You will see the below interface. ![!Jupyter Notebook Landing Page](images/jupyter_landing_page.png)
+
+5. When the job has been created, on the My Interactive Sessions page, click the button in front of Host (usually colored blue) in the format >_c0000.
+
+    ![!host image](images/cheaha_shell_button.png)
+
+    This should open into a Shell CLI as shown below.
+
+    ![!Cheaha Shell CLI](images/cheaha_shell_cli.png)
+
+6. In this interface, you can create, and activate environments, as well as install packages, modules and libraries into your activated environment.
+
+## How do we create a custom environment for PyTorch and TensorFlow
+
+The instructions below, provide a recommended step by step guide to creating and activating an environment that has PyTorch and/or TensorFlow installed and ready to use for deep learning projects.
+
+## Steps to Install PyTorch Using Cheaha CLI
```

Comment:
I think this heading should be one level deeper, so use `###`, because it is a subtopic of the previous heading.